rust/src/message/verdict.rs

```diff
@@ -13,9 +13,9 @@ pub enum Verdict {
     /// No malicious content found.
     Clean,
     /// Malicious content found.
-    Malicious,
+    Malicious(String),
     /// Potentially unwanted content found.
-    Pup,
+    Pup(String),
```

Comment:
We discussed this internally: If we add fields to the enum here, we would prefer a struct as member such that we could add more data about detections in the future (e.g. more details like which files are infected in an archive). It also makes it easier to document what the string is.